edit-post/components/plugin-more-menu-item/plugin-sidebar-more-menu-item.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/element';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { MenuItemsItem } from '@wordpress/components';
+
+const PluginSidebarMoreMenuItem = ( { isSelected, icon, label, onClick } ) => (
+	<MenuItemsItem
+		icon={ isSelected ? 'yes' : icon }
+		isSelected={ isSelected }
+		label={ label }
```

Comment:
Aside: When testing an example, I intuitively expected this to be the `children` of the component, and had to circle back here to find out it wasn't.

Reply:
Yeah, will fix. I had the same thought the other day 😃